utility/importer/brave_importer.cc

```diff
@@ -61,6 +62,8 @@ void BraveImporter::StartImport(const importer::SourceProfile& source_profile,
   // The order here is important!
   bridge_->NotifyStarted();
 
+  ImportReferral();
```

Comment:
Please add a comment for future devs explaining why referral import is handled differently (does not check/require user choice) from all other import data types.